Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -97,6 +97,7 @@
 		13E264F840239C8C99865921 /* document_test.cc in Sources */ = {isa = PBXBuildFile; fileRef = AB6B908320322E4D00CC290A /* document_test.cc */; };
 		1465E362F7BA7A3D063E61C7 /* database_id_test.cc in Sources */ = {isa = PBXBuildFile; fileRef = AB71064B201FA60300344F18 /* database_id_test.cc */; };
 		146C140B254F3837A4DD7AE8 /* bits_test.cc in Sources */ = {isa = PBXBuildFile; fileRef = AB380D01201BC69F00D97691 /* bits_test.cc */; };
+		148EC47BA70CEB561D4957C1 /* bundle_spec_test.json in Resources */ = {isa = PBXBuildFile; fileRef = 24F2F48EDB1645DF4AA82EB4 /* bundle_spec_test.json */; };
```

Comment:
Is this related to the PR? If not, let's try to remove all of these from the PR.